subscriptions/views/widget_timeline.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+
+from django.http import JsonResponse
+from rest_framework.decorators import api_view
+
+from associations.models import Event, Page
+from associations.serializers import EventSerializer, PageSerializer
+
+
+@api_view(["GET"])
+def widget_timeline_view(request):
+    # show the next events
+    events = (
+        Event.objects.filter(
+            ends_at__gt=datetime.datetime.now(),
+            starts_at=datetime.datetime.now() + datetime.timedelta(hours=6),
```

Comment:
Did you forget a `__gt`? Because this condition is _very_ precise
Also, why 6 hours? I think we can show every event which has not started yet, a Mineur / Minette / Mineuse / female Mineur's life is full of surprise.